README.md

```diff
@@ -2,73 +2,250 @@
 
 The best way to handle your search bar documentation with [MeiliSearch](https://github.com/meilisearch/MeiliSearch).
 
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-<!-- You can install de TOC plugin on VSCode and other text editors -->
+![docs-searchbar.js example](/assets/docs-searchbar-example.png)
 
+If you use Vuepress for your documentation, you might check out our [Vuepress plugin for MeiliSearch](https://github.com/meilisearch/vuepress-plugin-meilisearch).
+
+## Table of Contents <!-- omit in toc -->
+
+- [Installation and Usage](#installation-and-usage)
+  - [Before using the library](#before-using-the-library)
+  - [Basic Usage](#basic-usage)
+  - [Customization](#customization)
 - [Development Workflow](#development-workflow)
-  - [Requirements](#requirements)
-  - [Install](#install)
-  - [Build](#build)
-  - [Serve](#serve)
-  - [Test](#test)
-  - [Release](#release)
-- [Contributing](#contributing)
 - [Credits](#credits)
 
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+## Installation and Usage
+
+### Before using the library
+
+To integrate a search bar to your documentation and use this library, you need to scrap your website contents and to push it into a MeiliSearch instance.
+
+Hopefully, we provide all the tools needed and [guide you into these different steps](https://docs.meilisearch.com/resources/howtos/search_bar_for_docs.html) 🚀
+
+As a first introduction, you might only want to test this library without connecting it to your website.<br>
```

Comment:
```suggestion
As a first step, you might only want to test this library without connecting it to your website.<br>
```

Reply:
Since it is only for discovering purpose, I think introduction is best suited.